pnpm-lock.yaml

```diff
@@ -533,6 +533,9 @@ devDependencies:
   copy-webpack-plugin:
     specifier: ^10.2.4
     version: 10.2.4(webpack@5.88.0)
+  css-loader:
```

Comment:
should revert these changes from lock file and package.json.

Reply:
Sorry, I thought I got all of them.  I just revered the files to what was in master.

Reply:
OK, sorry for all of the commits, was having issues with my branches being out of sync.  package and pnpm-lock are both revered.  We should be good now.